### PR TITLE
remove create > invalid user settable id > too short test because min allowed is 1

### PR DESCRIPTION
**Pull request by Yoshiji**
**TL;DR:** The `Create/invalid_user_settable_id/too_short` generated test is not following AIP-122 because single-letter ID is valid.

---

https://google.aip.dev/122#resource-id-segments mentions the regex to validate the user-provided ID:
```
^[a-z]([a-z0-9-]{0,61}[a-z0-9])?$
```

Which follows RFC-1034 rule for labels. Quoting https://www.rfc-editor.org/rfc/rfc1034 (ctrl+f "ARPANET host names")

> The labels must follow the rules for ARPANET host names.  They must
> start with a letter, end with a letter or digit, and have as interior
> characters only letters, digits, and hyphen.  There are also some
> restrictions on the length.  **Labels must be 63 characters or less.**

A valid label can be a single letter. The generated test `too short` for user settable ID is not feasible because trying to test with an ID with a length equal to 0 = empty = the server will fall back to system-generated ID:

> An API may allow the {resource}_id field have the [field_behavior](https://google.aip.dev/203) OPTIONAL, **and generate a system-generated ID if one is not specified.**

Source: https://google.aip.dev/133#user-specified-ids

### CODE REVIEW

**Comment by ericwenn** [comment]
Nice catch @Yoshiji! :star: LGTM

We follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) in this repository, so you'll have to update the commit message.

**Comment by Yoshiji** [comment]
@ericwenn I reworded the commit message.

**Review by ericwenn — APPROVED** [review]
